File: coding-ig/bar-chart/bar.py
n = int(input())
_input = list(map(int,input().strip().split()))
n = len(_input)
_max = max(_input)
for i in range(1,_max+1):
    for j in range(0,n):
        if (i >= _max-(_input[j]-1)):
            print("#", end=" ")
        else:
            print(".", end=" ")
    print("\n")

File: coding-ig/bar-chart/bar.R
n <- as.integer(readline(promp=""));
input <-as.integer(strsplit(readline(promp="")," ")[[1]]);
n <- length(input);
max_ <-max(input);
for (i in 1:max_){
  for (j in 1:n){
    if (i >= max_ - ((input[j])-1)){
      cat("# "); 
    }else{
      cat(". ");
    }
  }
  cat("\n");
  
}